inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/http/HttpBaseSource.java

```diff
@@ -117,10 +117,7 @@ public void configure(Context context) {
 
         topic = context.getString(ConfigConstants.TOPIC);
         attr = context.getString(ConfigConstants.ATTR);
-        Configurables.ensureRequiredNonNull(context, ConfigConstants.TOPIC, ConfigConstants.ATTR);
```

Comment:
How to handle the message if the topic is empty and not checked?

Reply:
> How to handle the message if the topic is empty and not checked?

Use the topic in the topics.properties file to handle the message, instead of using the topic field in the dataproxy-mulit-pulsar-http-example.conf file.

Reply:
![image](https://user-images.githubusercontent.com/54351417/189065488-9fe4a7b9-18bd-412d-ba42-5d5c8ea70b44.png)

Reply:
@xuesongxs, you're right, and from your reply, you've figured out how it works, thanks!

Looking forward to your more contributions!!